diagnostic.md

```diff
@@ -57,13 +65,15 @@ Please write a method that traverses from a list item (`li`) DOM node to the
 body DOM node, then hides it.
 
 <!-- your answer starts here -->
-
+$('li').parent('body').hide()
```

Comment:
I believe you need to go up another level here: `$('li').parent().parent().hide();`